app/features/accessibility.test.js

```diff
@@ -0,0 +1,60 @@
+import test from 'ava'
+import { setupPptrTab, teardownPptrTab, getActiveTool, changeMode } from '../../tests/helpers'
+
+test.beforeEach(setupPptrTab)
+
+const contrastValueSelector = `document.querySelector('visbug-ally').$shadow.querySelector("span[contrast]").textContent.trim()`
+
+test('Can be activated', async t => {
+  const {page} = t.context;
+  await changeMode({page, tool: 'accessibility'})
+
+  t.is(await getActiveTool(page), 'accessibility')
+  t.pass()
+})
+
+test('Can reveal color contrasts between html nodes and backgrounds', async t => {
+  const {page} = t.context;
+  await changeMode({page, tool: 'accessibility'})
+
+  await page.hover('.google-blue')
```

Comment:
i estimate this is where the tests are failing on Travis.. but i'm not 100%

Reply:
It was a silly mistake. I wrote tests based on [this commit](https://github.com/GoogleChromeLabs/ProjectVisBug/pull/388/commits/64a1496aac00741f220537b3a83db2f5293e758a), and before pushing, I forgot to rebase the latest commit. so I rebased it and pushed commits not double-checking tests.

[Turns out there are more variables to change name as `foreground`.](https://github.com/GoogleChromeLabs/ProjectVisBug/blob/c73c042426a239102d397517ca8742583fd6bc78/app/features/accessibility.js#L173-L174)
I fixed them and tested again. It works fine now.